src/albums.go

```diff
@@ -5,16 +5,17 @@ import (
 	"os"
 	"path/filepath"
 	"regexp"
+	"time"
 )
 
 // TODO : next feature
 // *  Add possibility to reload a directory or add new directory at the runtime
 
-// MonthPics , slice to store pic together for the same month
-type MonthPics map[int][]*Photo
+// WeeklyPics , slice to store pic together for the same month
+type WeeklyPics map[int][]*Photo
 
 // YearPics , slice to store pic together for the same year and month
-type YearPics map[int]MonthPics
+type YearPics map[int]WeeklyPics
```

Comment:
I change this, the monthly basis is nice but all operations later will be weekly base, so I change my organization to make it easier to search later.